README.md

```diff
@@ -96,6 +96,17 @@ Most of the Azure resources deployed in the prior steps will incur ongoing charg
 
 We have provided some sample deployment scripts that you could adapt for your own purposes while doing a POC/spike on this. Those scripts are found in the [inner-loop-scripts directory](./inner-loop-scripts). They include some additional considerations and may include some additional narrative as well. Consider checking them out. They consolidate most of the walk-through performed above into combined execution steps.
 
+## Preview features
+
+While this reference implementation tends to avoid _preview_ features of AKS to ensure you have the best customer support experience; there are some features you may wish to evaluate in pre-production clusters that augment your posture around security, manageability, etc. Consider trying out and providing feedback on the following. As these features come out of preview, this reference implementation may be updated to incorporate them.
+
+* [Azure RBAC for Kubernetes Authentication](https://docs.microsoft.com/azure/aks/manage-azure-rbac) - An extension of the Azure AD integration already in this reference implementation. Allowing you to bind Kubernetes authentication to Azure RBAC role assignments.
```

Comment:
are we listing all AKS preview feature in here or just those we'd want to incorporate in the future?

Reply:
This is heading off questions about "what about preview features" -- it's not really exhaustive or prioritized or scoped to anything.  Just happened to be "topical" in the side thread :)